examples/async.rs

```diff
@@ -62,7 +61,9 @@ fn main() {
         .and_then(|_| {
             println!("Port was removed.");
             Ok(())
-        });
+        })
+
+    }).map(|_| () ).map_err(|_| () );
```

Comment:
Is it possible to just do a `.then(|_| Ok())` instead of two maps?

Reply:
yeah, i can also move the maps into the `tokio::run`, it's just a clear way of showing that it's all getting thrown away